PyPortal_ObliqueStrategies/oblique.py

```diff
@@ -0,0 +1,44 @@
+"""
+This code will display a random strategy from strategies.py when the
+PyPortal screen is pressed. See the original Oblique Strategies
+by Brian Eno & Peter Schmidt here: https://www.enoshop.co.uk/product/oblique-strategies
+"""
+import os
+import random
+import board
+from strategies import strategies
+from adafruit_pyportal import PyPortal
+
+cwd = os.getcwd() # the current working directory (where this file is)
```

Comment:
this doesn't do what you think, use the original cwd assignment from other guides